### DISCUSSION TITLE
Power to "first LED"  and LED-strip

### DISCUSSION THREAD

**Discussion by DariBer** [awawa-dev/HyperHDR#280] [Q&A] [answered]
I read about skipping the Voltage Level Shifter and use a diod and one LED. I saw the pictures on HyperHDR blogg, there's one thing I don't understand. I can see that the one LED is powered from the ESP board, but the LED strip, how's that powered?

Anyone of these 3 is the one to go with?
![ws-levelshift4](https://user-images.githubusercontent.com/26138185/169927113-15e425ca-2c15-4aec-bbca-0991bd865189.png)

![ws-levelshift_ext-pwr](https://user-images.githubusercontent.com/26138185/169926204-49c74eb3-98a5-4e96-8d14-651167979784.png)

![ws-levelshift3](https://user-images.githubusercontent.com/26138185/169926237-758805b9-7600-4995-ae49-1bebc6d4d9cb.png)

**Comment by awawa-dev** [comment, marked as answer]
The first is definitely wrong as you cannot power the LED strip from the ESP device. The second is also wrong unless you separate ESP USB power supply from LED string power supply somehow but still the minus pole is wrong. So it's only the third option is left and it's looking good (use HyperSerial firmware with "first led disable", probably WLED can also disable first LED).

**Reply by DariBer** [reply]
Thank you for your help, you are so helpful, wander all questions people have. So cool, will def donate money to you for this project.
